setup.cfg

```diff
@@ -1,3 +1,52 @@
+[metadata]
+name = uproot
+author = Jim Pivarski
+author_email = pivarski@princeton.edu
+maintainer = Jim Pivarski
+maintainer_email = pivarski@princeton.edu
+description = ROOT I/O in pure Python and NumPy.
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/scikit-hep/uproot4
+download_url = https://github.com/scikit-hep/uproot4/releases
+license = BSD 3-clause
+
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    Intended Audience :: Information Technology
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Operating System :: MacOS
+    Operating System :: POSIX
+    Operating System :: Unix
+    Programming Language :: Python
+    Programming Language :: Python :: 2.6
```

Comment:
The `python_requires` rules 2.6 out, do we want to list it here? If you ask for the versions, it does show up: ![PyPI - Python Version](https://img.shields.io/pypi/pyversions/uproot) (or, alternatively, we could change `python_requires`?)

Reply:
The python_requires now only constrains 2.6, so we can leave this.